open_instruct/ppo_vllm_thread_ray_gtrl.py

```diff
@@ -908,6 +943,14 @@ def repeat_generator():
             n=args.number_samples_per_prompt,
             stop=args.stop_strings,
         )
+        evaluation_generation_config = SamplingParams(
+            temperature=0.001,
```

Comment:
why do this?

Reply:
For reproducibility and better comparison? If we are using e.g., 0.7 temp, then even the same model would generate different resposnes.

Reply:
oh, but why 0.001 and not just greedy decode?

Reply:
Good point. I thought we couldn't specify zero but the docs indicate otherwise. Have just changed it.

<img width="787" alt="image" src="https://github.com/user-attachments/assets/b1a35132-03ea-48c1-8622-6d059eb7774e" />